modules/EnemyCounter.lua

```diff
@@ -55,7 +55,7 @@ function TMW_ST:CountInRange(stop)
     for i = 1, EnemyCounter.max_scan do
         name = 'nameplate' .. i
 
-        if _UnitExists(name) and _UnitReaction(name, "player") < 5 and _CheckInteractDistance(name, 2) == true then
+        if _UnitExists(name) and _UnitReaction(name, "player") < 5 and IsItemInRange("item:34368", name) == true then
```

Comment:
can you share where you took this from? I understand what you're trying to fix but I'm always hesitant of adding things like this (eg - what's this item and why is it a better indicator for melee range)?

Reply:
it's from LibRangeCheck-2.0
https://github.com/WeakAuras/LibRangeCheck-2.0/blob/7d716f7264e463a05471c00a1c36f3cdadaadbe8/LibRangeCheck-2.0/LibRangeCheck-2.0.lua#L321

Reply:
I think a better solution then would just be to use LibRangeCheck directly. iirc it's part of TMW anyway